lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -82,6 +128,75 @@ def self.serialize(store:, name:, serializer: nil)
         store_accessor store, name
       end
 
+      def buffers
+        pm_storage_adapter.buffers
+      end
+
+      def self.create_later(attrs, storage_adapter)
+        element = new(attrs)
+        storage_adapter.buffers[:upsert][element.unique_identifier] = element
+
+        # TODO: Why did we have to do this again? Don't get it
+        # keys_to_ignore = %i[unique_identifier created_at updated_at] #FIXME
+        # already_pending = persistence_elements[:to_upsert].find do |elt|
+        #   elt.class == self && attrs.symbolize_keys.except(*keys_to_ignore).all? { |k,v| elt.send(k).to_json == v.to_json }
+        # end
+        # if already_pending
+        #   already_pending
+        # else
+        #   to_create = new(attrs)
+
+        #   elements_to_create << to_create
+        #   to_create
+        # end
+      end
+
+      def self.assign_later(parent:, child:, buffer:)
+        buffer.merge!(parent => child)
```

Comment:
I feel like we can legitimately have multiple children for the same parent in the same buffer, which this looks like it doesn't support.